Projects/Foundations/Schemas/Docs-Topics/C/Community/Community-Workspaces/community-workspaces-013-api-data-fetcher-bot-demo-plugin-workspace.json

```diff
@@ -26,7 +26,7 @@
             "translations": [
                 {
                     "language": "RU",
-                    "text": "Разработчик",
+                    "text": "Разра��отчик",
```

Comment:
Does only github not know the letters marked with two question marks or superalgos too?

Reply:
That's really weird. I only edited the English in SA and used docs.save....so I don't know why the Russian translation was affected here.

Reply:
Should be sorted now as i committed the change a couple hours ago. Not sure why it still shows here but it does have the outdated tag so I guess it's overwritten?